.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+registries:
+
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+    day: "sunday"
+    time: "04:00"
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: org.jetbrains.kotlinx:kotlinx-coroutines-core
+    versions:
+    - "*-native-mt"
```

Comment:
Ikke helt sikker på att denne virker https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#specifying-dependencies-and-versions-to-ignore

Reply:
Dette virket ikke så fjernet den.. :(